Source/Core/VideoCommon/UberShaderPixel.cpp

```diff
@@ -672,303 +674,302 @@ ShaderCode GenPixelShader(APIType ApiType, const ShaderHostConfig& host_config,
+                 "          break;\n"
+                 "        }}\n"
+                 "\n"
+                 "        if (shift >= 0)\n"
+                 "          indtevtrans = indtevtrans >> shift;\n"
+                 "        else\n"
+                 "          indtevtrans = indtevtrans << ((-shift) & 31);\n"
+                 "      }}\n"
+                 "\n"
+                 "      // Wrapping\n"
+                 "      uint sw = {};\n",
+                 BitfieldExtract("tevind", TevStageIndirect().sw));
+    out.WriteFmt("      uint tw = {}; \n", BitfieldExtract("tevind", TevStageIndirect().tw));
+    out.WriteFmt(
         "      int2 wrapped_coord = int2(Wrap(fixedPoint_uv.x, sw), Wrap(fixedPoint_uv.y, tw));\n"
         "\n"
-        "      if ((tevind & %du) != 0u) // add previous tevcoord\n",
+        "      if ((tevind & {}u) != 0u) // add previous tevcoord\n",
         1 << TevStageIndirect().fb_addprev.StartBit());
-    out.Write("        tevcoord.xy += wrapped_coord + indtevtrans;\n"
-              "      else\n"
-              "        tevcoord.xy = wrapped_coord + indtevtrans;\n"
-              "\n"
-              "      // Emulate s24 overflows\n"
-              "      tevcoord.xy = (tevcoord.xy << 8) >> 8;\n"
-              "    }\n"
-              "    else if (texture_enabled)\n"
-              "    {\n"
-              "      tevcoord.xy = fixedPoint_uv;\n"
-              "    }\n"
-              "\n"
-              "    // Sample texture for stage\n"
-              "    if(texture_enabled) {\n"
-              "      uint sampler_num = %s;\n",
-              BitfieldExtract("ss.order", TwoTevStageOrders().texmap0).c_str());
-    out.Write("\n"
-              "      float2 uv = (float2(tevcoord.xy)) * " I_TEXDIMS "[sampler_num].xy;\n");
-    out.Write("      int4 color = sampleTexture(sampler_num, float3(uv, %s));\n",
-              stereo ? "float(layer)" : "0.0");
-    out.Write("      uint swap = %s;\n",
-              BitfieldExtract("ss.ac", TevStageCombiner().alphaC.tswap).c_str());
-    out.Write("      s.TexColor = Swizzle(swap, color);\n");
-    out.Write("    } else {\n"
-              "      // Texture is disabled\n"
-              "      s.TexColor = int4(255, 255, 255, 255);\n"
-              "    }\n"
-              "\n");
+    out.WriteFmt("        tevcoord.xy += wrapped_coord + indtevtrans;\n"
+                 "      else\n"
+                 "        tevcoord.xy = wrapped_coord + indtevtrans;\n"
+                 "\n"
+                 "      // Emulate s24 overflows\n"
+                 "      tevcoord.xy = (tevcoord.xy << 8) >> 8;\n"
+                 "    }}\n"
+                 "    else if (texture_enabled)\n"
+                 "    {{\n"
+                 "      tevcoord.xy = fixedPoint_uv;\n"
+                 "    }}\n"
+                 "\n"
+                 "    // Sample texture for stage\n"
+                 "    if(texture_enabled) {{\n"
```

Comment:
Do we care about spacing in here? Not like the linter can catch it inside a string 🤷 
```suggestion
                 "    if (texture_enabled) {{\n"
```